### PR TITLE
Add type declaration file for typescript

### PR DESCRIPTION
**Pull request by devholic**
Hello, I added type declaration file for typescript :D

### CODE REVIEW

**Comment by tsuyoshiwada** [comment]
Hi @devholic , Thanks you for PR !
Please wait for a while as i will check tomorrow :smiley:

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Component } from "react";
+
+export interface MDSpinnerProps {
```
Comment (tsuyoshiwada) [inline src/index.d.ts]: Greatest works! but, `ssrBehavior()` does not exist.
Can you add type definition of `ssrBehavior()` ?

* [API](https://github.com/tsuyoshiwada/react-md-spinner#ssrbehavior)
* [Source code](https://github.com/tsuyoshiwada/react-md-spinner/blob/master/src/ssr-behavior.js)

Comment (devholic) [inline src/index.d.ts]: Sure! :D

Comment (tsuyoshiwada) [inline src/index.d.ts]: Thanks !  Please add type definition :+1:

Comment (devholic) [inline src/index.d.ts]: Sorry for late 😭
I added namespace for `ssrBehavior` :D

Comment (tsuyoshiwada) [inline src/index.d.ts]: Sorry for late me too :cry:

I need some fix, but I will work and release it.
Thank you for the best work :muscle: !!

**Comment by devholic** [comment]
Thanks! :D

**Pull request merged**